<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Credit</class>
 <widget class="QDialog" name="Credit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>855</width>
    <height>767</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>855</width>
    <height>767</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>855</width>
    <height>767</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(50,50,50);</string>
  </property>
  <widget class="QLineEdit" name="sum_credit">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>30</y>
     <width>140</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>140</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>130</width>
     <height>30</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Pixeloid Sans</family>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#000;background-color:rgb(180, 179, 161);margin-top:3px;border:none;</string>
   </property>
   <property name="text">
    <string>1000000</string>
   </property>
   <property name="maxLength">
    <number>255</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="term">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>70</y>
     <width>140</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>140</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>130</width>
     <height>30</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Pixeloid Sans</family>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#000;background-color:rgb(180, 179, 161);margin-top:3px;border:none;</string>
   </property>
   <property name="text">
    <string>36</string>
   </property>
   <property name="maxLength">
    <number>255</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="percent">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>110</y>
     <width>140</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>140</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>130</width>
     <height>30</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Pixeloid Sans</family>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#000;background-color:rgb(180, 179, 161);margin-top:3px;border:none;</string>
   </property>
   <property name="text">
    <string>8</string>
   </property>
   <property name="maxLength">
    <number>255</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_1_sum_credit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>30</y>
     <width>220</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>40</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Eurostile</family>
     <pointsize>19</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(170,170,170);</string>
   </property>
   <property name="text">
    <string>Сумма кредита, руб.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2_credit_term">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>70</y>
     <width>220</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>40</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Eurostile</family>
     <pointsize>19</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(170,170,170);</string>
   </property>
   <property name="text">
    <string>Срок кредита, мес.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3_percent">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>110</y>
     <width>220</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>40</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Eurostile</family>
     <pointsize>19</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(170,170,170);</string>
   </property>
   <property name="text">
    <string>Процентная ставка, %</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="annuity">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>30</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Eurostile</family>
     <pointsize>17</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;</string>
   </property>
   <property name="text">
    <string>Аннуитетный</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="differential">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>60</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Eurostile</family>
     <pointsize>17</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;</string>
   </property>
   <property name="text">
    <string>Дифференцированный</string>
   </property>
  </widget>
  <widget class="QPushButton" name="payment_schedule">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>110</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>680</width>
     <height>4800</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Eurostile</family>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border-radius: 6px;
	color: white;
	background-color: rgb(126, 44, 61); 
}
QPushButton:pressed {
	background-color: rgb(146, 64, 81); 
}
QPushButton:hover {
	border: 1px solid rgb(146, 64, 81); 
}</string>
   </property>
   <property name="text">
    <string>График платежей</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="info_message">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>160</y>
     <width>650</width>
     <height>66</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>650</width>
     <height>66</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>650</width>
     <height>66</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Kefa</family>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
background-color: rgb(50,50,50);
border:none;</string>
   </property>
   <property name="cursorWidth">
    <number>0</number>
   </property>
  </widget>
  <widget class="QTableView" name="table_pay_shedule">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>811</width>
     <height>500</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>100</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>500</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderHighlightSections">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
